### PR TITLE
Add pyxenon

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by jan-janssen** [comment]
@conda-forge/help-python

File: recipes/pyxenon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyxenon-{{ version }}.tar.gz
+  sha256: 6840771716974da68d69e5cc24e017698f8f9bcebcfaf3cb5da41555d5ec772b
+  patches:
+    - setupunix.patch  # [unix]
+    - setupwin.patch  # [win]
+
```
Comment (jan-janssen) [inline recipes/pyxenon/meta.yaml:14]: The `pyxenon` package could be `noarch: python` but windows complains about the `bin/xenon-grpc` bash script and causes the build to fail. So I only copy the operation system specific shell scripts.

**Comment by jan-janssen** [comment]
I tried this before over a year ago - finally everything is working https://github.com/conda-forge/staged-recipes/pull/20245

**Review by xhochy — APPROVED** [review]

**Pull request merged**